CODEOWNERS

```diff
@@ -11,8 +11,8 @@
 # Packages
```

Comment:
not sure why but GitHub is giving this error: 
```
Unknown owner on line 4: make sure the team @superfluid-finance/fte exists and has write access to the repository
* @superfluid-finance/fte
* ```

Reply:
My guess is that `@superfluid-finance/fte` doesn't have write access?

Reply:
right, let me change it [protocol-devs](https://github.com/orgs/superfluid-finance/teams/protocol-devs)